extension/src/test/suite/index.ts

```diff
@@ -2,15 +2,7 @@ import * as path from 'path';
 import * as Mocha from 'mocha';
 import * as glob from 'glob';
 
-import * as fs from 'fs-extra';
-import * as baseConfig from "@istanbuljs/nyc-config-typescript";
-
-import 'ts-node/register';
-import 'source-map-support/register';
-
-const NYC = require('nyc');
-
-export async function run(): Promise<void> {
+export function run(): Promise<void> {
```

Comment:
this is overwritting Sen's previous change. Please merge first.

Reply:
OK, I will take a look